bucket/librewolf.json

```
@@ -22,6 +21,10 @@
     "checkver": {
         "url": "https://gitlab.com/api/v4/projects/13852981/releases",
         "jsonpath": "$[0].assets.links[1].url",
-        "regex": "/uploads/(?<randid>[a-z0-9]+)/librewolf-([\\d.]+)\\.en-US\\.win64\\.zip"
+        "regex": "/uploads/(?<randid>[a-z0-9]+)/librewolf-(?<version>[\\d.]+)\\.en-US\\.win64\\.zip"
+    },
+    "autoupdate": {
+        "url": "https://gitlab.com/librewolf-community/browser/windows/uploads/$matchRandid/librewolf-$matchVersion.en-US.win64.zip",
+        "extract_dir": "librewolf-$matchVersion"
```

```suggestion
        "url": "https://gitlab.com/librewolf-community/browser/windows/uploads/$matchRandid/librewolf-$version.en-US.win64.zip",
        "extract_dir": "librewolf-$version"
```

For version, we don't need to prefix `match`